desktop/src/main.rs

```diff
@@ -109,6 +110,10 @@ struct Opt {
     #[clap(long, short, default_value = "show-all")]
     scale: StageScaleMode,
 
+    /// Audio volume as a number between 0 (muted) and 1 (full volume)
+    #[clap(long, short, default_value = "1.0")]
+    volume: f32,
```

Comment:
Thinking out loud - Does it make more sense to define this as an integer in range 0-100? Feels a bit more user-friendly to me.

Reply:
You would need to also make that change for the `volume` property of the player on web, and think about how to do that without causing unexpected behavior in people's existing sites... let's leave this PR as-is